tck/src/main/java/io/smallrye/opentelemetry/tck/BaseTests.java

```diff
@@ -390,6 +393,59 @@ public void testUrlDoesNotExists()
         Assert.assertEquals(spans.size(), 0);
     }
 
+    @Test
+    @RunAsClient
+    private void testClient() throws URISyntaxException, MalformedURLException {
```

Comment:
Instead of using the client in the server endpoint `PATH_NESTED` could we just simply create the client in the test and make a request? I want to simplify this by executing the request directly in the test.

Then we need these tests:
* make a request and assert that correct tags are preset
* make sure parent span is linked to client span created by the client
* make a request and assert that context is put into the request headers.
* async test https://github.com/opentracing-contrib/java-jaxrs/blob/master/opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractClientTest.java#L170
* multi-threaded test https://github.com/opentracing-contrib/java-jaxrs/blob/master/opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractClientTest.java#L207 - please check if the client can be used by multiple threads